content/post/azure-monitor-terraform.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Azure Monitor & Terraform: Collecting custom log files"
+date: 2023-10-03T14:46:23+01:00
+hero: /images/hero-3.jpg
+excerpt: "Working with Azure Monitor in collecting custom log files using Terraform"
+timeToRead: 5
+authors:
+  - Dylan McGinley
+draft: true
+license: cc-by
+---
+To collect custom log files from a VM (Virtual Machine) managed by Azure, it is recommended to use Azure Monitor and the Azure Monitor Agent (AMA). The Azure Monitor Agent is the successor to the Log Analytics Agent which is due to be deprecated on August 31, 2024. There are currently issues associated with the azurerm provider in relation to Azure Monitor.
+
+I suspect in the future these issues will be addressed with updates but for now this post will serve to provide a workaround along with an explanation as to what these are.
+
+Azurerm version 3.74.0
+
+The issue
+Currently the advised method of collecting logs and storing them with Azure Monitor relies on the following configuration
```

Comment:
Make sure you run markdown-lint (included in Visual Studio Code's markdown extension) and confirm formatting on the rendered page using hugo.

This above isn't valid markdown, for instance. `The issue` is not a heading, and the lack of a blank newline would normally mean the paragraph is continued (i.e. the rendered paragraph would become _"The issue Currently the advised…"_, although the treatment of newlines kind of depends on the markdown variant.